c2corg_ui/static/js/apiservice.js

```diff
@@ -380,6 +387,37 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
 
 
 /**
+ * @param {undefined | string} token
+ * @param {string} lang
+ * @param {boolean} isProfile
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
+ */
+app.Api.prototype.readFeed = function(token, lang, isProfile) {
+  var url;
+  var params = {'pl': lang};
+  if (token) params['token'] = token;
```

Comment:
Readability and code consistency is more important than saving lines. And closure will make the lines optimisation when building the JS.
Whatever I won't fight for this :)